internal/backend/temporalclient/client.go

```diff
@@ -75,10 +79,19 @@ func New(cfg *Config, z *zap.Logger) (Client, error) {
 }
 
 func (c *impl) startDevServer(ctx context.Context, cfg *Config, opts client.Options) error {
-	cfg.DevServer.ClientOptions = &opts
+	temporalLog := path.Join(xdg.DataHomeDir(), "temporal_dev.log")
```

Comment:
not sure, but maybe we should make it configurable?

Reply:
I general I prefer to keep configuration minimal. Every option you add is something you need to carry on and to test. If you have only 10 true/false option you need to test 1024 combinations. Unless there's a user requirement I prefer to keep it this way.